src/_styles/_menu.scss

```diff
@@ -1,5 +1,54 @@
-.logo {
-  background: $grey-60;
-  height: 100px;
-  width: 100px;
+.menu {
+  font-size: $size-unit * 4; // 16px
+  width: 100%;
+  display: none;
+
+  .menu-items {
+    .menu-link {
+      display: flex;
+    }
+  }
+
+  @media (min-width: $breakpoint-sm) {
+    display: block;
+    .menu-wrap {
+      display: flex;
+      justify-content: flex-end;
+      background-color: $white;
+      width: 100%;
+
+      .menu-items {
+        display: flex;
+        align-content: center;
+        margin-left: $size-unit * 8; // 32px
+
+        &:hover {
+          border-bottom: $border-unit solid $dark-blue;

```

Comment:
Let's use here box-shadow instead of border. `box-shadow: inset 0 -1px 0 0 #224a54;` This will prevent from jumping menu element on hover.

Reply:
- [ ]  Done